BeeSwift/Settings/SettingsViewController.swift

```diff
@@ -87,6 +90,19 @@ class SettingsViewController: UIViewController {
     func showLogsTapped() {
         self.navigationController?.pushViewController(LogsViewController(), animated: true)
     }
+    
+    @objc private func fetchUser() {
+        Task { [weak self] in
```

Comment:
There's some interesting nuance to lifetimes here.

It might be tempting to do this:
```
          guard let self = self else { return }
```
But it's not quite right, as the view might be dismissed while waiting for the refreshUser call to complete, and in that case we don't want to hold on to `self`. 

Maybe worth a comment making this clear?